internal/cmd/kafka/command_link_describe.go

```diff
@@ -3,12 +3,24 @@ package kafka
 import (
 	"github.com/spf13/cobra"
 
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/kafkarest"
 	"github.com/confluentinc/cli/internal/pkg/output"
 )
 
+type describeOut struct {
+	Name                 string `human:"Name" serialized:"link_name"`
+	TopicName            string `human:"Topic Name" serialized:"topic_name"`
+	SourceClusterId      string `human:"Source Cluster" serialized:"source_cluster_id"`
+	DestinationClusterId string `human:"Destination Cluster" serialized:"destination_cluster_id"`
```

Comment:
Why the inconsistency here with the human output not containing "ID"?

Reply:
Sounds good, might be worth some investigation to see if other commands do this too